compose/desktop/desktop/samples-material3/src/jvmMain/kotlin/androidx/compose/desktop/examples/material3/SwitchRippleAnimationBug.kt

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.compose.desktop.examples.material3
+
+import androidx.compose.material3.Switch
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+
+@Composable
+fun SwitchRippleAnimationBug() {
```

Comment:
Later it would be cool to write a test on it!

Reply:
This is a bug in animation. I think we should check it again after merging with stable material3 inside androidx. (Maybe it will be fixed)

But, yes. We should add tests for material3 components as well.